aws/resource_aws_ecs_task_definition.go

```diff
@@ -197,6 +197,37 @@ func resourceAwsEcsTaskDefinition() *schema.Resource {
 										Optional: true,
 										Default:  "/",
 									},
+									"transit_encryption": {
+										Type:     schema.TypeString,
+										ForceNew: true,
+										Optional: true,
```

Comment:
add validation that string is one of  EFSTransitEncryption enums

Reply:
Updated